kubernetes_state/tests/test_kubernetes_state.py

```diff
@@ -379,3 +379,23 @@ def test_pod_phase_gauges(aggregator, instance, check):
     aggregator.assert_metric(
         NAMESPACE + '.pod.status_phase', tags=['namespace:default', 'phase:Failed', 'optional:tag1'], value=2
     )
+
+
+def test_telemetry(aggregator, instance, check):
+    instance['telemetry_enable'] = True
+    instance['_text_filter_blacklist'] = ['resourcequota']
+
+    endpoint = instance['kube_state_url']
+    scraper_config = check.config_map[endpoint]
+
+    # this would be normally done in the __init__ function of the check
+    scraper_config['telemetry_enable'] = instance['telemetry_enable']
+    scraper_config['_text_filter_blacklist'] = instance['_text_filter_blacklist']
```

Comment:
You should recreate the check, it would catch the bug I saw about about the config variable naming.